docs/source/daml/intro/8_Dependencies.rst

```diff
@@ -0,0 +1,86 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+8 Working with Dependencies
+===========================
+
+The application from Chapter 7 is a complete and secure model for atomic swaps of assets, but there is plenty of room for improvement. However, one can't implement all feature before going live with an application so it's important to understand way to change already running code. There are fundamentally two types of change one may want to make:
+
+1. Upgrades, which change existing logic. For example, one might want the ``Asset`` template to have multiple signatories.
+2. Extensions, which merely add new functionality though additional templates.
+
+Upgrades are covered in their own own section outside this introduction to DAML: :doc:`/upgrade/index` so in this section we will extend the chapter 7 model with a simple second workflow: A multi-leg trade. In doing so, you'll learn about:
+
+- The software architecture of the DAML Stack
+- Dependencies and Data Dependencies
+- Identifiers
+
+Since we are extending chapter 7, the setup for this chapter is slightly more complex:
+
+#. In a base directory, load the chapter 7 project using ``daml new 7Composing --template daml-intro-7``. The directory ``7Composing`` here is important as it'll be referenced by the other project we are creating.
+#. In the same directory, load the chapter 8 project using ``daml new 8Dependencies --template daml-intro-8``.
```

Comment:
There is no template called `daml-intro-8`.

Reply:
There should be in ``daml-head`` from this branch.

Reply:
No, you have `daml-intro-8-assets` and `daml-intro-8-upgrade` but no `daml-intro-8`.

Reply:
I did in the previous version which was about upgrading, but I changed that. I'm puzzled why you still see the old templates.

Reply:
duh, I forgot to pull :facepalm: